### PR TITLE
Style/update readme

### PR DESCRIPTION
**Pull request by Nchappui**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,63 @@
...
+
+
+The organization of the group went as follows:
+
+We had a [scrum board](https://github.com/Ph0tonic/SDP_Projet/projects/1) were we would write issues of the app or potential features to be added and each week we would assign our self to tasks to have an estimated working time of 8 hours per week. Once a task would be assigned, it would be moved in the `Sprint tasks` meaning that somebody is working on it and once the task would be completed and merged on the master branch, it would be moved in the `Done` column of the corresponding week. We had in total 13 weeks to complete our app. We also had one or two more meetings per week to each discuss about our progression and potential roadblocks.
+Once a team member had finished his task, he had to create a pull request so that other team members could see what he did and accept his merge request or ask for changes.
```
Comment (Ph0tonic) [inline README.md:19]: Accept missing a `s`

File: README.md [hunk 2]
```diff
@@ -1,4 +1,63 @@
...
+
+Since the development of our app was part of a course at EPFL, we had to meet some requirements that were:
+
+### Correctness & Utility:
+Our app has many user-friendly features implemented as well as a nice UI to easily understand how to use the app.
+
+### Split app model: 
+We used Google Firebase to create the searching teams and each team has its own members stored in the firebase, once the heatmap is created, it is also stored in the firebase.
```
Comment (Ph0tonic) [inline README.md:27]: Coma after `teams`

File: README.md [hunk 3]
```diff
@@ -1,4 +1,63 @@
...
+It also stores the last map attributes such as the zoom or the position.
+
+### Offline mode: 
+We can download parts of the map so that it stays on the phone, and we can use them while being offline.
+
+### Testing:
+We tested most of our methods and have a current code coverage of [![Test Coverage](https://api.codeclimate.com/v1/badges/198be56409634587df0f/test_coverage)](https://codeclimate.com/github/Ph0tonic/SDP_Projet/test_coverage)
+
+## Working without a drone
+To be  able to use our application without having a drone, the user must download a simulator as well as other useful tools, all of it is explained in our [wiki](https://github.com/Ph0tonic/SDP_Projet/wiki)
```
Comment (Ph0tonic) [inline README.md:46]: Two spaces between `be` and `able`

File: README.md [hunk 4]
```diff
@@ -1,4 +1,63 @@
...
+
+The application works as follows: Once you are in a search group, the operators can select a zone on the map and the drone will choose an optimized path in this area to collect signal strengths from the potential victim. Once the drone gets the first signal, it starts drawing a live heatmap on the map that can be seen from both the operators and the rescuers. Any member of the group can then place a special marker on the map on the potential location of the victim.
+
+
+The organization of the group went as follows:
+
+We had a [scrum board](https://github.com/Ph0tonic/SDP_Projet/projects/1) were we would write issues of the app or potential features to be added and each week we would assign our self to tasks to have an estimated working time of 8 hours per week. Once a task would be assigned, it would be moved in the `Sprint tasks` meaning that somebody is working on it and once the task would be completed and merged on the master branch, it would be moved in the `Done` column of the corresponding week. We had in total 13 weeks to complete our app. We also had one or two more meetings per week to each discuss about our progression and potential roadblocks.
```
Comment (Ph0tonic) [inline README.md:18]: Change `per week to each discuss about our progression and potential roadblocks.` into `per week to discuss our progression and potential roadblocks.`

File: README.md (hunk 1 above)
Comment (Nchappui) [reply README.md:19]: it is correct like that

File: README.md (hunk 2 above)
Comment (Nchappui) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (Nchappui) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (Nchappui) [reply README.md]: done

**Pull request merged**